frontend/src/pages/admin/AdminEinzelnerSchueler/AdminFacharbeitEntryPage.tsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * Übersichtsseite für die Facharbeit.
+ */
+
+import React, { useState } from 'react';
+import AdminFacharbeitApplicationForm from './AdminFacharbeitApplicationForm';
+import AdminFacharbeitStudentListPage from './AdminFacharbeitStudentListPage';
+
+function AdminFacharbeitEntryPage() {
+  const [role] = useState('student');
```

Comment:
bei einer Reacthook brauchst du immer ein Variable und ein setter, das sieht dann so aus: 
const [role, setRole] = useState('student');

der defaultvalue, also jetzt wo dort 'student' drinsteht sollte leer sein (bzw. leerer string), und über eine useEffect Hook initial gesetzt werden. Die useEffect Hook wird grundsätzlich einmal ausgeführt wenn die Funktionale Komponente geladen wird und kann dann sie noch abhängig von verschiedenen werten, die im dependency array stehen, ausgeführt werden. Der Syntax sieht so aus:

  useEffect(() => {
// hier code der ausgeführt werden soll
  }, [ dependency array]);

in der useEffect kannst du dann die role aus dem redux state lesen und mit "setRole(*roleAusReduxState*); setzen